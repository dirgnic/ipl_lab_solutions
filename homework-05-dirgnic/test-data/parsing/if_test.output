3:1:
  |
3 | <empty line>
  | ^
unexpected end of input
expecting "if", "let", or "return"
