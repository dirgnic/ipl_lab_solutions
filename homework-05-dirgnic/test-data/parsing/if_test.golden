Seq [Let "x" (Num 0) (Res (Num 0))] (Ite (Sma (Var "x") (Num 0)) (Res (Num 1)) (Res (Num 2)))
